rust/matches/src/lib.rs

```diff
@@ -161,6 +147,23 @@ pub mod matches {
         match_instance.minimum_allowed_entry_time = minimum_allowed_entry_time;
         match_instance.leave_allowed = leave_allowed;
 
+        msg!("namespaces");
+        if desired_namespace_array_size > 0 {
```

Comment:
Feel like I've seen this code in a couple places now. Does it make sense to pop out into a utility method to DRY out?

Reply:
Its once in all the different crates, we need a commons crate and then put this in there